print("please")
# 오류 해결